src/main/MQ2ImGuiConsole.cpp

```diff
@@ -27,1504 +27,1638 @@
+		if (fontstr != consoleDebugName)
+		{
+			s_consoleFontName = ImGui::GetFont()->GetDebugName();
+			WritePrivateProfileString("MQConsole", "ConsoleFontOnStartup", s_consoleFontName, mq::internal_paths::MQini);
+			s_consoleFontSize = (int)ImGui::GetFontSize();
+			WritePrivateProfileInt("MQConsole", "ConsoleFontSize", s_consoleFontSize, mq::internal_paths::MQini);
+		}
+		ImGui::NewLine();
+
+		ImGui::NewLine();
+		if (ImGui::Button("Clear Saved Console Settings"))
+		{
+			s_consoleVisibleOnStartup = false;
+			WritePrivateProfileBool("MQConsole", "ShowMacroQuestConsole", s_consoleVisibleOnStartup, mq::internal_paths::MQini);
+			s_isConsoleLocked = false;
+			WritePrivateProfileBool("MQConsole", "LockConsoleWindow", s_isConsoleLocked, mq::internal_paths::MQini);
+			s_isTitleBarShown = false;
+			WritePrivateProfileBool("MQConsole", "NoTitleBar", s_isTitleBarShown, mq::internal_paths::MQini);
+			s_alphaConsoleNormal = 100;
+			WritePrivateProfileInt("MQConsole", "AlphaNormalOnStartup", s_alphaConsoleNormal, mq::internal_paths::MQini);
+			s_alphaConsoleFade = 100;
+			WritePrivateProfileInt("MQConsole", "AlphaFadeOnStartup", s_alphaConsoleFade, mq::internal_paths::MQini);
+			s_consoleFontSize = 13;
+			WritePrivateProfileInt("MQConsole", "ConsoleFontSize", s_consoleFontSize, mq::internal_paths::MQini);
+			s_consoleFontName = "lucon.ttf, 13px";
+			s_consoleFontSize = 13;
+			int fontNumber = FontNumberFromString(s_consoleFontName);
+			ImGui::GetIO().FontDefault = ImGui::GetIO().Fonts->Fonts[fontNumber];
+
+			WritePrivateProfileString("MQConsole", "ConsoleFontOnStartup", s_consoleFontName, mq::internal_paths::MQini);
+		}
 	}
-}
 
-void InitializeImGuiConsole()
-{
-	s_consoleVisibleOnStartup = GetPrivateProfileBool("MacroQuest", "ShowMacroQuestConsole", false, mq::internal_paths::MQini);
-	s_consoleVisible = s_consoleVisibleOnStartup;
-	if (gbWriteAllConfig)
+	void InitializeImGuiConsole()
 	{
-		WritePrivateProfileBool("MacroQuest", "ShowMacroQuestConsole", s_consoleVisibleOnStartup, mq::internal_paths::MQini);
-	}
+		s_consoleVisibleOnStartup = GetPrivateProfileBool("MQConsole", "ShowMacroQuestConsole", false, mq::internal_paths::MQini);
+		s_consoleVisible = s_consoleVisibleOnStartup;
+		s_alphaConsoleFade = GetPrivateProfileInt("MQConsole", "AlphaFadeOnStartup", 100, mq::internal_paths::MQini);
+		s_alphaConsoleNormal = GetPrivateProfileInt("MQConsole", "AlphaNormalOnStartup", 100, mq::internal_paths::MQini);
+		s_consoleFontSize = GetPrivateProfileInt("MQConsole", "ConsoleFontSize", 13, mq::internal_paths::MQini);
+		s_isTitleBarShown = GetPrivateProfileBool("MQConsole", "NoTitleBar", false, mq::internal_paths::MQini);
+		s_isConsoleLocked = GetPrivateProfileBool("MQConsole", "LockConsoleWindow", false, mq::internal_paths::MQini);
+
+		// load saved font 
+		s_consoleFontName = GetPrivateProfileString("MQConsole", "ConsoleFontOnStartup", "Lucon.ttf, 13px", mq::internal_paths::MQini);
```

Comment:
the default here is capitalized, but FontNumberFromString is using a case sensitive comparison.